components/html-parser.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+interface HtmlParserProps {
+  children: string;
+}
+
+export function HtmlParser(props: HtmlParserProps) {
+  const { children } = props;
+  return (
+    <span
+      dangerouslySetInnerHTML={{
+        __html: children.replace(/(?:\r\n|\r|\n)/g, "<br>"),
+      }}
+    ></span>
+  );
+}
```

Comment:
I think we should be able to use [htmr](https://github.com/pveyes/htmr) for this without sacrificing that much performance. Can you confirm Mas @zainfathoni?

Reply:
Yes, we can use `htmr` in our project.

Initialle I intended to use it, but I decided to let it go because I didn't understand clearly its benefits.

Since one of its benefits are clear enough now, let's try to use `htmr` instead of implementing our own tag-escaping logic. 😉